src/submission/submission-process.service.ts

```diff
@@ -798,6 +794,25 @@ export class SubmissionProcessService {
     this.logger.debug('Setting common template context parameters with ', {submissionEmailParamsDto});
     await this.setCommonParams(submissionEmailParamsDto);
 
+    //Get the recipients list from the recipient's list API
+    submissionEmailParamsDto.ccEmail = await this.recipientListService.getEmailRecipients(
```

Comment:
Why is this ccEmail instead of toEmail?

Reply:
Hi Dwayne, the short answer is that it is not clear from the requirement.  Your comment on ticket 6080 from July 25 (comment #1) implies that the response from this ticket should be used as the cc value.  

I also tried to clarify with Mark that very question on ticket 6183.  Please see the ticket comments (https://github.com/US-EPA-CAMD/easey-ui/issues/6183).  Here is question #5 form this ticket comments:

- I will be using the email list returned from the recipient list api as the 'cc' email address. I am setting the 'to' to the value of camdecmpsaux.submission_set.user_email value. Please confirm if this is the intention.

Based on the above two, I assumed the value was to be used as the cc value.  The 'to' is being set to camdecmpsaux.submission_set.user_email value.

Reply:
It looks like you are correct Yonatan.